composer.json

```diff
@@ -20,48 +20,63 @@
     ],
     "require": {
         "php":                                  ">=5.3.3",
-
+        "a2lix/translation-form-bundle":        "~2.0",
+        "doctrine/collections":                 "~1.2",
+        "doctrine/common":                      "~2.3",
         "doctrine/doctrine-bundle":             "~1.3",
         "doctrine/doctrine-cache-bundle":       "~1.0",
+        "doctrine/doctrine-migrations-bundle":  "2.1.*@dev",
+        "doctrine/migrations":                  "~1.0@dev",
         "doctrine/orm":                         "~2.4",
-        "doctrine/migrations":                  "dev-master",
-        "doctrine/doctrine-migrations-bundle":  "dev-master",
-        "friendsofsymfony/rest-bundle":         "~1.5@dev",
+        "doctrine/phpcr-odm":                   "1.2.2",
+        "friendsofsymfony/elastica-bundle":     "~3.0.2",
+        "friendsofsymfony/oauth-server-bundle": "~1.4",
+        "friendsofsymfony/rest-bundle":         "1.5.*@dev",
         "friendsofsymfony/user-bundle":         "2.0.*@dev",
         "hwi/oauth-bundle":                     "~0.3",
         "incenteev/composer-parameter-handler": "~2.0",
-        "jackalope/jackalope-doctrine-dbal":    "1.1.*",
+        "jackalope/jackalope":                  "1.1.4",
+        "jackalope/jackalope-doctrine-dbal":    "1.1.2",
         "jms/serializer-bundle":                "~0.13",
         "jms/translation-bundle":               "1.1.*",
-        "knplabs/knp-gaufrette-bundle":         "*@dev",
+        "knplabs/gaufrette":                    "0.2.*@dev",
+        "knplabs/knp-gaufrette-bundle":         "0.2.*@dev",
+        "knplabs/knp-menu-bundle":              "1.1.*",
         "knplabs/knp-snappy-bundle":            "*@dev",
         "liip/imagine-bundle":                  "~1.0",
         "mathiasverraes/money":                 "*@dev",
-        "payum/payum":                          "~0.12.0",
-        "payum/payum-bundle":                   "~0.12.0",
-        "omnipay/stripe":                       "~2.0",
+        "omnipay/omnipay":                      "~2.3",
+        "payum/payum":                          "~0.12",
+        "payum/payum-bundle":                   "~0.12",
+        "prezent/doctrine-translatable":        "1.0.*@dev",
         "sensio/distribution-bundle":           "2.3.*",
         "stof/doctrine-extensions-bundle":      "1.1.*",
-        "symfony-cmf/block-bundle":             "~1.0",
-        "symfony-cmf/content-bundle":           "~1.0",
-        "symfony-cmf/menu-bundle":              "~1.0",
-        "symfony-cmf/routing-bundle":           "~1.0",
-        "symfony-cmf/create-bundle":            "~1.0",
-        "symfony-cmf/media-bundle" :            "~1.0",
+        "swiftmailer/swiftmailer":              "~5.0",
+        "sylius/cart-bundle":                   "0.13.*@dev",
```

Comment:
Why was this added here? :) Unfortunately it's preventing me from installing forks for testing new features.